_raml/apps/products/examples/get_product_productid_crossselling_response.json

```diff
@@ -224,6 +224,11 @@
         "isCustomizable": true,
         "customizableTextLength": 50,
         "customizableHeadlineText": "Customize your product",
+        "newnessDate": "2023-08-08T00:00:00Z",
```

Comment:
Not necessarily, since in the MBO you can save a date, with it marked as new. But then turn 'marked as new' off, the date will still be saved in case 'marked as new' is turned on again.